src/STEMMUS_SCOPE.m

```diff
@@ -327,6 +333,10 @@
             k = NoTime(KT);
         end
         %%%%% Updating the state variables. %%%%%%%%%%%%%%%%%%%%%%%%%%%%
+	% Update Groundwater variables (received from BMI, added by Mostafa)
+	headBotmLayer = GroundwaterSettings.headBotmLayer; 
+	indexBotmLayer = GroundwaterSettings.indexBotmLayer;        
+        
```

Comment:
```suggestion 
        
```

Reply:
Still have couple of questions here:

1- For the line `GroundwaterSettings = io.readGroundwaterSettings()`, do I need to add it at 
- only at line [249](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/4a34628429f61cda017b89cf0325f488777168e9/src/STEMMUS_SCOPE.m#L249)
- only at line [303](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/4a34628429f61cda017b89cf0325f488777168e9/src/STEMMUS_SCOPE.m#L303)
- at both lines [249](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/4a34628429f61cda017b89cf0325f488777168e9/src/STEMMUS_SCOPE.m#L249) and line [303](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/4a34628429f61cda017b89cf0325f488777168e9/src/STEMMUS_SCOPE.m#L303)

Reply:
As discussed only at line 249 in the initialization part.